### ISSUE TITLE
Lch description sounds weird

### ISSUE THREAD

**Issue by AleksandrSl** [evilmartians/oklch-picker#41]
Description:
[Unlike HSL](https://wildbit.com/blog/accessible-palette-stop-using-hsl-for-color-systems), OKLCH always has predictable contrast after color transformation.
But Lab is LCH have [hue shift](https://lch.evilmartians.io/#35,55,297,100) on chroma changes. [Oklab and OKLCH](https://oklch.evilmartians.io/#45.8,0.143,285.98,100) were created to solve this problem.

What's weird:
* Start form OKLCH description, maybe intentional
* `But Lab is LCH have` seems grammatically wrong. Not sure what was supposed to be here. Maybe `and`

**Comment by ai** [comment]
> Start form OKLCH description, maybe intentional

Yes, we need a place for link and do not have enough space for proper wording.

> But Lab is LCH have seems grammatically wrong. Not sure what was supposed to be here. Maybe and

Thanks. Fixed 9b8531c
